paper.md

```diff
@@ -1,33 +1,39 @@
 ---
-title: 'Separation using ChatGPT'
+title: 'Self-refining of Pseudo Labels for Music Source Separation with Noisy Labeled Data'
 tags:
   - separation
-  - u-net
+  - self-refining
 authors:
-  - name: Fabian-Robert Stoter^[co-first author] # note this makes a footnote saying 'co-first author'
-    orcid: 0000-0003-0872-7098
-    affiliation: "1, 2" # (Multiple affiliations must be quoted)
-  - name: Author Without ORCID^[co-first author] # note this makes a footnote saying 'co-first author'
+  - name: Junghyun Koo^[co-first author] # note this makes a footnote saying 'co-first author'
+    orcid: 0009-0004-4468-0367
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Yunkee Chae^[co-first author] # note this makes a footnote saying 'co-first author'
+    orcid: 0009-0000-8073-3754
     affiliation: 2
-  - name: Author with no affiliation^[corresponding author]
-    affiliation: 3
+  - name: Chang-Bin Jeon
+    affiliation: 1
+  - name: Kyogu Lee^[corresponding author]
+    orcid: 0000-0002-4210-0312
+    affiliation: "1, 2, 3"
 affiliations:
- - name: Audioshake
+ - name: Department of Intelligence and Information
    index: 1
- - name: Institution Name
+ - name: Interdisciplinary Program in Artificial Intelligence
    index: 2
- - name: Independent Researcher
+ - name: Artificial Intelligence Institute, Seoul National University
    index: 3
-date: 10 September 2023
+date: 04 October 2023
 bibliography: paper.bib
-arxiv-doi: 10.21105/joss.01667
+arxiv-doi: 10.48550/arXiv.2307.12576
 ---
 
 # Abstract
 
-The abstract should be around 250 words long as usual, and should be provided
-for submissions of all categories: posters, long talks and discussions.
-
-* Please briefly describe the particular points your submission focuses on and/or the
-problems it aims to solve.
-* For all categories except discussions, please also briefly summarize the contributions.
+Music source separation (MSS) faces challenges due to the limited availability of crrectly-labeled individual instrument tracks.
+This paper proposes and automated approach for refining mislabeled instrument tracks in a partially noisy-labeled dataset.
+Similar yet different from self-training [@xie2020self], our approach learns directly from noisy labeled data and re-labeles the training data.
```

Comment:
```suggestion
Similar, yet different from self-training [@xie2020self], our approach learns directly from noisy labeled data and re-labeles the training data.
```

Reply:
Hello, @faroit.
Thank you for pointing out the typo.
We corrected the typo and committed it again.